### ISSUE TITLE
[BUG] - Cannot open two Item Piles from actor list

### ISSUE THREAD

**Issue by Ikabodo** [fantasycalendar/FoundryVTT-ItemPiles#58]
**Describe the bug**
If you open an item pile from the actor list, then try to open another, the second one does not open. It throws an error in the console:
```js
item-pile-inventory.js:60 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'uuid')
[Detected 1 package: item-piles]
    at item-pile-inventory.js:60:68
    at Array.filter (<anonymous>)
    at Function.getActiveAppFromPile (item-pile-inventory.js:58:52)
    at Function.show (item-pile-inventory.js:87:37)
```

I am not sure if this is intended or not. But it is throwing an error so I assume it's not intended. :)

**To Reproduce**
1. Click Item Pile in actor list
2. Click another Item Pile in actor list
3. (Second sheet does not show)

**Expected behavior**
Two sheets should open, or no error should show. Having to sheet's is preferable, this way you can transfer items between piles. (party storage chests for example)

**Setup:**
 - Foundry Version: 9.249
 - Item Piles Version: 1.4.0
 - OS: Windows 10
 - Browser: Chrome

**Active modules:**
 - Item Piles
 - libwrapper
 - socketlib

**Comment by Haxxer** [comment]
Thank you for the report, this is a known issue that I'll address in 1.4.1

**Comment by Haxxer** [comment]
Fixed in 1.4.1
